### PR TITLE
cleanup(rules): adjust output fields wrt new style guide

### PR DESCRIPTION
**Pull request by incertum**
https://falco.org/docs/rules/style-guide/

**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

> /kind feature

> /kind bug

/kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

/area rules

> /area registry

> /area build

> /area documentation

**Proposed rule [maturity level](https://github.com/falcosecurity/rules/blob/main/CONTRIBUTING.md#maturity-levels)**

> Uncomment one (or more) `/area <>` lines (only for PRs that add or modify rules):

> /area maturity-stable

> /area maturity-incubating

> /area maturity-sandbox

> /area maturity-deprecated

**What this PR does / why we need it**:

@leogr please review with https://github.com/falcosecurity/falco-website/pull/1094 and let me know if we agree on these fields? Same for the name to the left of the field, e.g. `cmdline` or `command` consistently?
That's why I kept this PR to the first ~10 rules only.

In addition, how many rules can I update wrt to output in one PR going forward?

Idea would be to first list the fields more specific to the rule and then append the agreed upon standard fields. The reviewer should double check that no critical unique fields were removed in the process.

The cleanup would also involve removing all `proc.pid` fields, I am explaining in https://github.com/falcosecurity/falco-website/pull/1094 why those fields are best kept for customization by adopters. This being said we have yet to improve the UX to support adding output fields generically to each rule for example ... as we discussed in several Falco tickets.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by leogr** [comment]
See https://github.com/falcosecurity/falco-website/pull/1094#pullrequestreview-1550223503

**Comment by leogr** [comment]
> @leogr please review with [falcosecurity/falco-website#1094](https://github.com/falcosecurity/falco-website/pull/1094) and let me know if we agree on these fields? Same for the name to the left of the field, e.g. `cmdline` or `command` consistently?

Done

File: rules/falco_rules.yaml [hunk 1]
```diff
@@ -2749,8 +2672,7 @@
     (modify_shell_history or truncate_shell_history) and
        not var_lib_docker_filepath and
        not proc.name in (docker_binaries)
-  output: >
-    Shell history had been deleted or renamed (user=%user.name user_loginuid=%user.loginuid type=%evt.type command=%proc.cmdline pid=%proc.pid fd.name=%fd.name name=%evt.arg.name path=%evt.arg.path oldpath=%evt.arg.oldpath %container.info)
+  output: Shell history deleted or renamed (fd.name=%fd.name name=%evt.arg.name path=%evt.arg.path oldpath=%evt.arg.oldpath evt_type=%evt.type user=%user.name user_uid=%user.uid user_loginuid=%user.loginuid process=%proc.name proc_exepath=%proc.exepath parent=%proc.pname command=%proc.cmdline terminal=%proc.tty exe_flags=%evt.arg.flags %container.info)
```
Comment (leogr) [inline rules/falco_rules.yaml:2675]:
```suggestion
  output: Shell history deleted or renamed (fd_name=%fd.name name=%evt.arg.name path=%evt.arg.path oldpath=%evt.arg.oldpath evt_type=%evt.type user=%user.name user_uid=%user.uid user_loginuid=%user.loginuid process=%proc.name proc_exepath=%proc.exepath parent=%proc.pname command=%proc.cmdline terminal=%proc.tty exe_flags=%evt.arg.flags %container.info)
```

File: rules/falco_rules.yaml [hunk 2]
```diff
@@ -2762,8 +2684,7 @@
   condition: >
     ((spawned_process and proc.name in (shred, rm, mv) and proc.args contains "bash_history") or
      (open_write and fd.name contains "bash_history" and evt.arg.flags contains "O_TRUNC"))
-  output: >
-    Shell history had been deleted or renamed (user=%user.name user_loginuid=%user.loginuid type=%evt.type command=%proc.cmdline pid=%proc.pid fd.name=%fd.name name=%evt.arg.name path=%evt.arg.path oldpath=%evt.arg.oldpath exe_flags=%evt.arg.flags %container.info)
+  output: Bash history deleted or renamed (fd.name=%fd.name name=%evt.arg.name path=%evt.arg.path oldpath=%evt.arg.oldpath evt_type=%evt.type user=%user.name user_uid=%user.uid user_loginuid=%user.loginuid process=%proc.name proc_exepath=%proc.exepath parent=%proc.pname command=%proc.cmdline terminal=%proc.tty exe_flags=%evt.arg.flags %container.info)
```
Comment (leogr) [inline rules/falco_rules.yaml:2687]:
```suggestion
  output: Bash history deleted or renamed (fd_name=%fd.name name=%evt.arg.name path=%evt.arg.path oldpath=%evt.arg.oldpath evt_type=%evt.type user=%user.name user_uid=%user.uid user_loginuid=%user.loginuid process=%proc.name proc_exepath=%proc.exepath parent=%proc.pname command=%proc.cmdline terminal=%proc.tty exe_flags=%evt.arg.flags %container.info)
```

File: rules/falco_rules.yaml [hunk 3]
```diff
@@ -3401,8 +3278,7 @@
   condition: >
     open_write and container and (fd.name=/proc/self/exe or fd.name startswith /proc/self/fd/) and not docker_procs and not proc.cmdline = "runc:[1:CHILD] init"
   enabled: false
-  output: >
-    Detect Potential Container Breakout Exploit (CVE-2019-5736) (user=%user.name process=%proc.name file=%fd.name cmdline=%proc.cmdline pid=%proc.pid %container.info)
+  output: Detect Potential Container Breakout Exploit (CVE-2019-5736) (file=%fd.name evt_type=%evt.type user=%user.name user_uid=%user.uid user_loginuid=%user.loginuid process=%proc.name proc_exepath=%proc.exepath parent=%proc.pname command=%proc.cmdline terminal=%proc.tty exe_flags=%evt.arg.flags %container.info)
```
Comment (leogr) [inline rules/falco_rules.yaml:3281]:
```suggestion
  output: Detect Potential Container Breakout Exploit (CVE-2019-5736) (fd_name=%fd.name evt_type=%evt.type user=%user.name user_uid=%user.uid user_loginuid=%user.loginuid process=%proc.name proc_exepath=%proc.exepath parent=%proc.pname command=%proc.cmdline terminal=%proc.tty exe_flags=%evt.arg.flags %container.info)
```

File: rules/falco_rules.yaml [hunk 4]
```diff
@@ -3414,9 +3290,7 @@
   condition: >
     open_read and container and (fd.name glob /proc/*/environ)
     and not proc.name in (known_binaries_to_read_environment_variables_from_proc_files)
-  output: >
-    Environment variables were retrieved from /proc files (user=%user.name user_loginuid=%user.loginuid program=%proc.name
-    command=%proc.cmdline pid=%proc.pid file=%fd.name parent=%proc.pname gparent=%proc.aname[2] ggparent=%proc.aname[3] gggparent=%proc.aname[4] container_id=%container.id image=%container.image.repository)
+  output: Environment variables were retrieved from /proc files (file=%fd.name gparent=%proc.aname[2] ggparent=%proc.aname[3] gggparent=%proc.aname[4] evt_type=%evt.type user=%user.name user_uid=%user.uid user_loginuid=%user.loginuid process=%proc.name proc_exepath=%proc.exepath parent=%proc.pname command=%proc.cmdline terminal=%proc.tty exe_flags=%evt.arg.flags %container.info)
```
Comment (leogr) [inline rules/falco_rules.yaml:3293]:
```suggestion
  output: Environment variables were retrieved from /proc files (fd_name=%fd.name gparent=%proc.aname[2] ggparent=%proc.aname[3] gggparent=%proc.aname[4] evt_type=%evt.type user=%user.name user_uid=%user.uid user_loginuid=%user.loginuid process=%proc.name proc_exepath=%proc.exepath parent=%proc.pname command=%proc.cmdline terminal=%proc.tty exe_flags=%evt.arg.flags %container.info)
```

File: rules/falco_rules.yaml [hunk 5]
```diff
@@ -3059,11 +2963,7 @@
     not network_local_subnet and
     k8s.ns.name in (namespace_scope_network_only_subnet)
   enabled: false
-  output: >
-    Network connection outside local subnet
-    (command=%proc.cmdline pid=%proc.pid connection=%fd.name user=%user.name user_loginuid=%user.loginuid container_id=%container.id
-     image=%container.image.repository namespace=%k8s.ns.name
-     fd.rip.name=%fd.rip.name fd.lip.name=%fd.lip.name fd.cip.name=%fd.cip.name fd.sip.name=%fd.sip.name)
+  output: Network connection outside local subnet (fd.rip.name=%fd.rip.name fd.lip.name=%fd.lip.name fd.cip.name=%fd.cip.name fd.sip.name=%fd.sip.name connection=%fd.name lport=%fd.lport rport=%fd.rport fd_type=%fd.type fd_proto=fd.l4proto evt_type=%evt.type user=%user.name user_uid=%user.uid user_loginuid=%user.loginuid process=%proc.name proc_exepath=%proc.exepath parent=%proc.pname command=%proc.cmdline terminal=%proc.tty exe_flags=%evt.arg.flags %container.info)
```
Comment (leogr) [inline rules/falco_rules.yaml:2966]:
```suggestion
  output: Network connection outside local subnet (fd_rip_name=%fd.rip.name fd_lip_name=%fd.lip.name fd_cip_name=%fd.cip.name fd_sip_name=%fd.sip.name connection=%fd.name lport=%fd.lport rport=%fd.rport fd_type=%fd.type fd_proto=fd.l4proto evt_type=%evt.type user=%user.name user_uid=%user.uid user_loginuid=%user.loginuid process=%proc.name proc_exepath=%proc.exepath parent=%proc.pname command=%proc.cmdline terminal=%proc.tty exe_flags=%evt.arg.flags %container.info)
```

**Review by leogr — COMMENTED** [review]
First pass

File: rules/falco_rules.yaml (hunk 2 above)
Comment (incertum) [reply rules/falco_rules.yaml:2687]: @leogr in the style guide we said `file=%fd.name`? Go with that consistently, aka fix this but to `file`?

**Review by leogr — APPROVED** [review]

File: rules/falco_rules.yaml (hunk 4 above)
Comment (LucaGuerra) [reply rules/falco_rules.yaml:3293]: I think `file` is used consistently, while if we have a connection the `fd.name `output reads `connection`. the original makes sense to me.

**Pull request merged**